enginetest/script_queries.go

```diff
@@ -1141,6 +1141,35 @@ var ScriptTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "WHERE clause considers ENUM/SET types for comparisons",
+		SetUpScript: []string{
+			"CREATE TABLE test (pk BIGINT PRIMARY KEY, v1 ENUM('a', 'b', 'c'), v2 SET('a', 'b', 'c'));",
+			"INSERT INTO test VALUES (1, 2, 2);",
```

Comment:
This test should really have >=2 rows so that you verify that the new filter logic is actually matching appropriately